src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/systemv/start-rpm-template

```diff
@@ -38,7 +38,7 @@
 
 exec="${{chdir}}/bin/${{exec}}"
 prog="${{app_name}}"
-lockfile=/var/lock/subsys/$prog
+lockfile="/var/lock/subsys/${{app_name}}"
```

Comment:
Interessting. This leads  to the rpmlint error, because rpm doesn't substitute the variables?